class ValidationError(Exception):
    pass


class ArgumentsError(Exception):
    pass
